EIPS/eip-3300.md

```diff
@@ -0,0 +1,66 @@
+---
+eip: 3300
+title: Phase out refunds
+author: William Morriss (@wjmelements)
+discussions-to: https://ethereum-magicians.org/t/eip-3300-phase-out-refunds/5434
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-02-26
+replaces: 3298
```

Comment:
```suggestion
```
You cannot replace a non-final EIP.  Also, the "replaces" is generally not used for Core EIPs since each hard fork is an upgrade from a previous state.